syntax = "proto2";

import "api/metadata/game_metadata.proto";
import "api/stats/game_stats.proto";
import "api/stats/events.proto";
import "api/player.proto";

package api;

message Game {
    optional api.metadata.GameMetadata game_metadata = 1;
    repeated Player players = 2;
    repeated api.stats.Hit hits = 3;
    optional api.stats.GameStats game_stats = 4;
    optional bytes kickoff_frames = 5;
    optional bytes game_frames = 6;
}
